.github/workflows/benchmark.yml

```diff
@@ -1,76 +0,0 @@
-on:
-  push:
-    # Sequence of patterns matched against refs/tags
-    branches:
-      - "develop"
-      - "master"
-    tags:
-      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
-  pull_request:
-    branches: ["develop"]
```

Comment:
This specific key under `pull_request` didn't get copied over. Is that okay?

Reply:
It means that we will run benchmarks on nested ors -- the results will be incorrect since the baseline data is coming from develop

Reply:
What are nested ors? Is there a good way around this?

Reply:
nested prs* eg if you make a PR into a branch other than master